src/components/AddEvent.js

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react'
+
+export default class AddEvent extends Component {
+    render() {
+        return (
+            <div>
+                <p>Add new event</p>
```

Comment:
Try and have a look at the Seeding to get a sense of what inputs are expected

Reply:
fixed